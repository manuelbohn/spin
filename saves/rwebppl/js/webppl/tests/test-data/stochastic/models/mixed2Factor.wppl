var model = function() {
  var x = flip();
  var std = x ? 0.5 : 3;
  var y = gaussian(1, std);
  // This factor shifts the means of the Gaussians from 1 to 0.
  factor((1 - 2 * y) / (2 * Math.pow(std, 2)));
  return y;
};
