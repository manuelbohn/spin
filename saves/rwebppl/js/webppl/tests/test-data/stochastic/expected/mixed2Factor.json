{
  "mean": 0,
  "std": 2.157
}
